content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md

```diff
@@ -133,7 +133,10 @@ As a node operator, you should upgrade to the [latest released version of Corda]
 
 In this release:
 
-* Corda Shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. You can read more about using the standalone shell [here](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
+* Corda shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. For more information about using the standalone shell, see [The standalone shell](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
+{{< note >}} 
+The Corda shell, which has a dependency on Groovy, is a standalone product since Corda Enterprise Edition 4.9. As a result, for any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.
```

Comment:
It looks like this note is now in the 4.9 Release Notes section?
If so, the first sentence is probably redundant as you've said in the previous section that the shell has been separated out and the release affected is implied by the section?